.github/workflows/docker.yml

```
@@ -6,12 +6,10 @@ on:
   pull_request:
     branches:
       - "**"
-    paths-ignore:
-      - '**/*.md'
-      - 'configuration/**'
-      - 'kubernetes/**'
-      - 'examples/**'
-      - 'scripts/**'
+    paths:
+      - 'docker/Dockerfile'
+      - 'Cargo.toml'
```

I now realized that maybe we want to check the `examples` as well?

```suggestion
      - 'Cargo.toml'
      - 'examples/Cargo.toml'
```

---

Do we? Because examples is copied in it's entirety so we don't care if its content changes. Only if we change the name of the examples directory does this break.